.github/workflows/ci.yml

```diff
@@ -28,9 +28,9 @@ jobs:
           ref: ${{ github.base_ref || github.ref_name }}
 
       - name: Set up Go 
-        uses: actions/setup-go@268d8c0ca0432bb2cf416faae41297df9d262d7f
+        uses: actions/setup-go@424fc82d43fa5a37540bae62709ddcc23d9520d4 # v2.1.5
```

Comment:
We don't want to go back that far, we want latest of 3.3.0 (hash `268d8c0ca0432bb2cf416faae41297df9d262d7f`)